packages/roosterjs-plugin-picker/lib/PickerPlugin.ts

```diff
@@ -199,7 +227,7 @@ export default class PickerPlugin<T extends PickerDataProvider = PickerDataProvi
         this.setAriaActiveDescendant(isSuggesting ? 0 : null);
     }
 
-    private handleKeyDownEvent(event: PluginKeyboardEvent) {
+    private handleKeyDownEvent(event: PluginDomEvent) {

```

Comment:
why need to change the type?

Reply:
Actually this name is confusing. It is not "handle" the event, but "cancel" the event. Need to use a better name

Reply:
This was changed because currently, the event can either be a PluginKeyboardEvent or PluginInputEvent (from Android). Though in the Android case, we probably don't need to keep track of `eventHandledOnKeyDown `.